index.html

```diff
@@ -68,7 +68,7 @@ <h2>I’am a Software Developer</h2>
     </div>
     <section class="work">
       <section class="cards">
-        <h1>My Recent Works</h1>
+        <h2>My Recent Works</h2>
         <hr />
         <div>
           <img src="image/firstImg.png" alt="" />
```

Comment:
- [ ] Please make sure that the page has a minimum contrast ratio of at least 4.5:1. The page currently has these contrast issues:
![image](https://user-images.githubusercontent.com/62290545/166463866-712489ce-4e9b-4153-8491-791ff39a096e.png)
![image](https://user-images.githubusercontent.com/62290545/166464038-93c39902-e949-4efe-bace-09cc5928c114.png)
![image](https://user-images.githubusercontent.com/62290545/166464137-d991c3f2-c5fe-4ccc-9b80-4ba1172031cd.png)

- You can check the color contrast ratio from [this website](https://www.aditus.io/button-contrast-checker/).